index.css

```diff
@@ -0,0 +1,130 @@
+* { box-sizing: border-box;}
+Body{
+  background-attachment: fixed;
+  background-image: url('http://www.fleming.kyschools.us/userfiles/316/Clipart%20Images/math.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin:auto;
+}
+.calculator-title{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.calculator-calcBody{
+  box-shadow: 30px 30px 30px grey;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
```

Comment:
This is one way to attempt center/center positioning. Checkout flexbox. It's a way better solution. 

```css
position: fixed;
top: 20%;
left:40%;
width: 20%;
height: 75%;
```

Also the height and width of the calculator is not relative to the page size. It should be a fixed pixel amount.